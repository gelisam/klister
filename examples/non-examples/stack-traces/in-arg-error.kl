#lang "prelude.kl"

(define fail (lambda (something) (+ something 1)))

(example (fail (error "in-arg-error!")))
