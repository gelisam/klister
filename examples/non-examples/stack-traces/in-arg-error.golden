Type mismatch at in-arg-error.kl:5.23-5.38. Expected Syntax but got String
